dev/html/src/imports/script-assert.js

```diff
@@ -92,6 +94,26 @@ window.Assert = {
             )
         }
     },
+    xTransformEquals: (element) => {
+        let style = element.style.transform
+        const computedStyle = window.getComputedStyle(element).transform
+
+        style = style.replace("translateX(", "").replace(")", "")
+
+        const matrixType = computedStyle.includes("3d") ? "3d" : "2d"
+
+        let x = 0
+        const matrixValues = computedStyle
+            .match(/matrix.*\((.+)\)/)[1]
+            .split(", ")
```

Comment:
I wonder if it would be better to construct a dom matrix with the string to avoid the parsing, although that does come with the cost of constructuring a matrix so not sure if it's a good idea 🤔

Reply:
It's just a test script so the cost isn't a big problem but likewise it can just be a bit shit